packages/main/src/plugin/recommendations/recommendations-api.ts

```diff
@@ -24,6 +24,7 @@ export interface ExtensionBanner {
   description: string;
   icon: string;
   thumbnail: string;
+  published?: string;
```

Comment:
With just the word `published` I expect this to be a boolean, like 'is it published or not?' I would need to find docs or an example to understand what to put here. Maybe should be `publishDate` or `embargoUntil` to more clearly state its purpose?

Reply:
Yes I agree, done ✅